packages/rx_bloc_list/example/lib/main.dart

```diff
@@ -164,7 +164,7 @@ class UserBloc extends $UserBloc {
   final _paginatedList = BehaviorSubject<PaginatedList<User>>.seeded(
     PaginatedList<User>(
       list: [],
-      pageSize: 50,
+      pageSize: 5,
```

Comment:
Please make sure you commit changes, which are only related to this ticket

Reply:
Not sure how this happened, but it's fixed now.